components/react/hooks/src/useMediaQuery/index.js

```diff
@@ -1,13 +1,16 @@
 import {useState, useEffect} from 'react'
 
-const supportMatchMedia =
-  typeof window !== 'undefined' && typeof window.matchMedia !== 'undefined'
-
 function useMediaQuery(queryInput, {defaultMatches = false} = {}) {
-  const query = queryInput.replace(/^@media( ?)/m, '')
+  const supportMatchMedia =
```

Comment:
```suggestion
  const hasMatchMedia =
```
boolean const must include `has` or `is` prefixes